Week2/AmirHossein/homework/pokemon-app/script.js

```diff
@@ -0,0 +1,49 @@
+window.onload = main;
+function main() {
+    const url = 'https://pokeapi.co/api/v2/pokemon/?limit=20&offset=20'
+    const btn = document.createElement('button');
+    btn.innerHTML = 'Get Pokemon!'
+    document.body.appendChild(btn);
+    const select = document.createElement('select');
+    document.body.appendChild(select);
+    btn.addEventListener('click', () => [
+        fetchData(url, select)
```

Comment:
this is strange! use `{}` to define the body of the callback instead of `[]`

Reply:
I think it was a typo mistake but it's strange that it was working!